application.py

```diff
@@ -47,6 +49,8 @@ def startup_event():
     json_logging.config_root_logger()
 
     log = logging.getLogger(__name__)
+
+    log.info(config.to_string())
```

Comment:
There is a more Pythonic way of providing a string representation. See https://www.journaldev.com/22460/python-str-repr-functions